client/src/app/site/deployment-center/deployment-center-setup/step-configure/configure-vsts-build/windows-frameworks/windows-frameworks.component.ts

```diff
@@ -93,32 +93,35 @@ export class WindowsFramworksComponent implements OnInit, OnDestroy {
     private _cacheService: CacheService,
     private _translateService: TranslateService
   ) {
-    this.wizard.buildSettings
-      .get('pythonSettings')
-      .get('framework')
-      .valueChanges.takeUntil(this._ngUnsubscribe$)
-      .subscribe(val => {
-        this.setupValidators(val);
-      });
+    this.setupValidators();
   }
 
   get getFramework() {
     return this.selectedPythonFramework;
   }
-  private setupValidators(val) {
+  private setupValidators() {
     this.requiredValidator = new RequiredValidator(this._translateService, false);
-    if (this.wizard.wizardValues.buildSettings.applicationFramework === WebAppFramework.Python && val === PythonFrameworkType.Django) {
-      this.wizard.buildSettings
-        .get('pythonSettings')
-        .get('djangoSettingsModule')
-        .setValidators([this.requiredValidator.validate.bind(this.requiredValidator)]);
-      this.wizard.buildSettings
-        .get('pythonSettings')
-        .get('djangoSettingsModule')
-        .updateValueAndValidity();
-    } else {
-      this.removeValidators();
-    }
+    this.wizard.buildSettings.get('applicationFramework').setValidators([this.requiredValidator.validate.bind(this.requiredValidator)]);
```

Comment:
fix indentation

Reply:
it is likely looking wrong in the pr but is actually correct.